source/documentation/05-payment-flow-overview

```diff
@@ -1,47 +1,48 @@
-# Payment flow overview
+# Payment flow 
 
 This section outlines how your service will interact with GOV.UK Pay after integration.
 
->This does not apply to users who use the [self service payments](/quick_start_guide//#self-service-payments) functionality.
+>This does not apply to users who use the [self-service payments](/quick_start_guide//#self-service-payments) functionality.
 
-## Overview of payment flow
+## Overview 
 
 When an end user needs to make a payment, your service makes an API call to create a new payment, then redirects the user to the payment pages hosted on GOV.UK Pay.
 
-The end user enters their payment details (for example, credit/debit card details and billing address) on the Pay pages. Pay handles all the details of verifying the payment with the underlying Payment Service Provider.
+The end user enters their payment details (for example, credit/debit card details and billing address) on the GOV.UK Pay pages. GOV.UK Pay verifies the payment with the underlying Payment Service Provider(PSP).
 
 After the transaction reaches a final state, the end user is then redirected back to your service.
 
 A final state means that the payment:
 
 + succeeds
-+ fails either because it is declined or the user chooses to cancel
++ fails either because it is declined 
++ fails because the user chooses to cancel
 + fails due to a technical error
 + fails because it is cancelled by your service
```

Comment:
can also fail because it times out - once a payment is created, users have 90 minutes to complete it